src/lib/env.js

```diff
@@ -45,7 +45,7 @@ if (process.env.VCAP_SERVICES) {
 }
 
 // gracefully output message and exit if any required config is undefined
-if (settings.cloudantEndpoint) {
+if (settings.cloudantEndpoint && settings.cloudantEndpoint.includes('/')) {
 	let tmp = settings.cloudantEndpoint;
 	settings.cloudantEndpoint = tmp.substring(tmp.indexOf('/') + 2);
 }
```

Comment:
1. The comment in line 47 seems out of place because this check doesn't output a message. Maybe move after.

2. The code behaves correctly, but maybe this is easier to read.  `settings.cloudantEndpoint = settings.cloudantEndpoint.replace('http://', '').replace('https://', '');`.  It should also make an invalid input like `cloudant.com/mydb` easier to identify.